po/pl.po

```diff
@@ -1885,25 +1885,44 @@ msgstr "darktable - błąd otwierania bazy danych"
 msgid "delete database"
 msgstr "skasuj bazę danych"
 
-#: ../src/common/database.c:2695
+#: ../src/common/database.c:2698
+msgid "click later to be asked on next startup"
+msgstr "kliknij „później”, aby zdecydować przy następnym uruchomieniu"
+
+#: ../src/common/database.c:2702
+msgid "click later to be asked when closing darktable"
+msgstr "kliknij „później”, aby zdecydować przy zamykaniu darktable"
+
+#: ../src/common/database.c:2706
+msgid "click later to be asked next time when closing darktable"
+msgstr "kliknij „później”, aby zdecydować przy następnym zamykaniu darktable"
+
+#: ../src/common/database.c:2710
```

Comment:
Correct, this would be the literal translation. But at the same time the final point here is to let users choose the time when they make the _decision_, and literal translation in this case sounds very unnatural to me. Hence my choice of wording.

Reply:
Hmmm... How on earth having written the code and the message AND being polish seems to me like I was supposed to be correct but your explaination makes more sense. You win. your version IS better, I just tried it and its sooo much better.

Reply:
@johnny-bit Really glad to hear that. :-) On a side note: One of the rules I like to use when translating (or preparing native messages) is "the software should make a simple, concise, but natural dialog with the user".

Reply:
Agreed. I hope my msg changes (in english) are in line with that.